frontend/src/routes/GymPage/index.tsx

```diff
@@ -25,6 +26,32 @@ const GymBarContainer = styled.div`
   top: 0;
 `
 
+const GymTabContainer = styled.div<{ isSelected?: boolean }>`
+  flex: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 5px;
+  background-color: #fff;
+  color: #191919;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
```

Comment:
@hui444 consider abstracting common stylings into "global" style varaibles so they wont need to repeat.